eventActions/accountabilityActions.js

```diff
@@ -25,7 +25,7 @@ class accountabilityActions {
 			const currentChannel = sentMessage.channel;
 
 			// Check if there are too many existing pins
-			currentChannel.fetchPinnedMessages().then(messages => {
```

Comment:
Good fix. This isn't related to your PR, but there might be other parts of the codebase using the old function that will need the same change. Just something to keep in mind